docs/user-guide.md

```diff
@@ -1889,6 +1889,17 @@ directory does not have to exist, but initial backups will fail if the
 destination is not available or writeable.</td>
 </tr>
 <tr>
+<td>GCS backup configuration</td>
```

Comment:
I think we might need some more documentation around the gcsfuse option, particularly what cloud-level prerequisites are required, and maybe even an example.